std-bits/table/src/main/java/org/enso/table/data/column/builder/ObjectBuilder.java

```diff
@@ -127,4 +128,15 @@ private void grow() {
   protected void resize(int desiredCapacity) {
     this.data = Arrays.copyOf(data, desiredCapacity);
   }
+
+  private AggregatedProblems preExistingProblems = AggregatedProblems.of();
+
+  public void setPreExistingProblems(AggregatedProblems preExistingProblems) {
+    this.preExistingProblems = preExistingProblems;
+  }
```

Comment:
This allows an ObjectBuilder after retyping to inherit any problems of the earlier builder.